src/main/java/uk/gov/hmcts/pdda/business/services/publicdisplay/DisplayConfigurationHelper.java

```diff
@@ -95,11 +102,19 @@ private static XhbCourtRoomDao[] getMultiSiteCourtRoomData(final Collection<?> v
     }
 
     private static boolean isCourtMultiSite(final Integer courtId,
-        final EntityManager entityManager) {
-        Optional<XhbCourtDao> dao = new XhbCourtRepository(entityManager).findById(courtId);
+        XhbCourtRepository xhbCourtRepository) {
+        Optional<XhbCourtDao> dao = xhbCourtRepository.findById(courtId);
         return dao.isPresent() && dao.get().getXhbCourtSites().size() > 1;
     }
 
+    // This one is called dynamically

```

Comment:
Why are you changing the src/main classes?

Reply:
Functionality has not changed in src/main only passing variables up from a parent method. Static methods won't accept mocking a variable unless it's passed into it. Wrapper methods have been placed here so that both existing code and test classes can still reference it. Running the app will confirm this.